crates/terminal/src/terminal.rs

```diff
@@ -333,6 +333,16 @@ impl TerminalBuilder {
             release_channel::AppVersion::global(cx).to_string(),
         );
 
+        // setup_env modifies the current process's environment, which we are
+        // ignoring in favor of the env variable here. Instead, we copy over
+        // the relevant variables from the current environment after
+        // modification so that we don't have to re-implement the logic here.
+        alacritty_terminal::tty::setup_env();
```

Comment:
Can we just not call setup_env at all? It's pretty fishy that we call it, and it does a bunch of file IO on the main thread. Maybe we can just set TERM and COLORTERM directly?

Reply:
how would you propose checking if alacritty is a valid terminfo entry on the system?

Reply:
I don't think we want to set our TERM to alacritty at all, it should probably be `zed`, right?